app/services/katello/pulp3/repository.rb

```diff
@@ -138,21 +138,6 @@ def get_remote(href = repo.remote_href)
         api.remotes_api.read(href)
       end
 
-      def remote_needs_updates?
-        if repo.remote_href
-          remote = get_remote
-          # The proxy auth creds are not returned by the Pulp API.
-          # The creds don't need to be checked here because they
-          # won't be updated outside of UpdateRemote.
-          computed = compute_remote_options.slice!(:proxy_username, :proxy_password)
```

Comment:
Just curious, why not remove username/password from computed rather than remove this whole method?

Reply:
Good question!  The goal was to update the remote if it needed an update, so we would fetch the remote compare it to what it should look like and then update if it didn't match.  

With the change in pulp that caused this, username and password are no longer included in the response, nor is proxy username or proxy password (from a few months ago), so now with 4 attributes that we can no longer detect changes for It seemed simpler and less error prone to just always update the remote no matter what.  Its also a relatively cheap operation, so it shouldnt' be an issue on sync.